scrape/scrape_test.go

```diff
@@ -196,17 +195,11 @@ func TestScrapePoolStop(t *testing.T) {
 	}
 
 	mtx.Lock()
-	if len(stopped) != numTargets {
-		t.Fatalf("Expected 20 stopped loops, got %d", len(stopped))
-	}
+	testutil.Equals(t, len(stopped), numTargets)
```

Comment:
Can you include the message here so we know which value was expected vs what the output was in the test, you can include a string after `numTargets`

Reply:
Good point, thanks! I've update the PR.